limitations/limitation.go

```diff
@@ -76,42 +75,42 @@ func ReadLimitations(file string) (map[string]*UserLimitations, error) {
 	return limitations, json.Unmarshal(bz, &limitations)
 }
 
-func GetLimitationExpiration(userID disgord.Snowflake, command string) (*time.Time, error) {
+func GetLimitationExpiration(userID string, command string) (*time.Time, error) {
```

Comment:
Replace `disgord.Snowflake` into `string` since `disgord.Snowflake` is the discord spec.